trade-finance-manager-ui/server/controllers/helpers/can-deal-be-cancelled.helper.ff-test.ts

```diff
@@ -0,0 +1,93 @@
+import { DEAL_SUBMISSION_TYPE, TEAM_IDS, TFM_DEAL_CANCELLATION_STATUS } from '@ukef/dtfs2-common';
+import { canDealBeCancelled } from './can-deal-be-cancelled.helper';
+import { TfmSessionUser } from '../../types/tfm-session-user';
+
+const pimUser = { teams: [TEAM_IDS.PIM] } as TfmSessionUser;
+const nonPimUser = { teams: [TEAM_IDS.UNDERWRITERS] } as TfmSessionUser;
+
+const { AIN, MIN, MIA } = DEAL_SUBMISSION_TYPE;
+const { COMPLETED, SCHEDULED, DRAFT } = TFM_DEAL_CANCELLATION_STATUS;
+
+describe('canDealBeCancelled', () => {
+  describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to true', () => {
```

Comment:
We have a lot of inconsistent test descriptions for feature flag tests:

```ts
describe('when deal cancellation is enabled', () => {
describe('when deal cancellation is disabled', () => {
  it('does not call next when deal cancellation is enabled', () => {

describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to true', () => {
describe('when FF_TFM_DEAL_CANCELLATION_ENABLED is disabled', () => {
describe('when FF_TFM_DEAL_CANCELLATION_ENABLED is enabled', () => {

it("redirects to '/not-found' if the feature flag is not enabled", () => {
it('calls the next function if the feature flag is enabled', () => {
```

Not sure what to suggest here, we should align these in a different PR

Reply:
yes good spot - seems like there's no clear precedent here so maybe one for another PR